### PR TITLE
feat: Adding the Brazilian portuguese translation to the app [Done]

### PR DESCRIPTION
**Pull request by andrenoberto**
Brazilian portuguese translation has bee finished!
Some terms are still in english because it makes more sense for us, since the literal translation will be something really wierd.

Glad to be contributing with this project. 😄

Best regards,
See you guys.

### CODE REVIEW

File: src/shared/locales/index.js [hunk 1]
```diff
@@ -8,6 +8,10 @@ export const availableLanguages = [
     value: 'en-US',
     label: 'English'
   },
+  {
+    value: 'pt-BR',
+    label: 'Português do Brasil'
```
[the remark this answers is not in the document]
Comment (andrenoberto) [reply src/shared/locales/index.js]: Sure, i'll made that change.

File: src/shared/locales/all.js [hunk 2]
```diff
@@ -16,6 +18,12 @@ const resources = {
       ...appLocaleEnUS
     }
   },
+  'pr-BT': {
```
Comment (agalwood) [inline src/shared/locales/all.js]: Please make sure you can switch languages successfully.
I find a typo: (
`pr-BT` => `pt-BR`

File: src/shared/locales/all.js [hunk 3]
```diff
@@ -1,9 +1,11 @@
 import eleLocaleEn from 'element-ui/lib/locale/lang/en'
+import eleLocalePtBr from 'element-ui/lib/locale/lang/pt-br'
```
Comment (agalwood) [inline src/shared/locales/all.js]: Please keep the variable naming case uniform.
`eleLocalePtBr` => `eleLocalePtBR`

File: src/shared/locales/all.js [hunk 4]
```diff
@@ -16,6 +18,12 @@ const resources = {
       ...appLocaleEnUS
     }
   },
+  'pr-BT': {
+    translation: {
+      ...eleLocalePtBr,
```
Comment (agalwood) [inline src/shared/locales/all.js]: Please keep the variable naming case uniform.
`eleLocalePtBr` => `eleLocalePtBR`

Comment (andrenoberto) [reply src/shared/locales/all.js]: Sorry. Fixed!

File: src/shared/locales/all.js (hunk 3 above)
Comment (andrenoberto) [reply src/shared/locales/all.js]: Sorry. Fixed!

File: src/shared/locales/all.js (hunk 2 above)
Comment (andrenoberto) [reply src/shared/locales/all.js]: Sorry. Fixed!

**Pull request merged**